core/src/test/java/edu/ucr/cs/riple/core/ArgumentNullableFlowTest.java

```diff
@@ -47,8 +47,9 @@ public void nullableFlowDetectionEnabledTest() {
             // DepA by 2, DepB by 0 and DepC by 1. Hence, the total effect is: -2.
             new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullable(int)"), -2),
             // Report below should appear due to flow of Nullable back to target by downstream
-            // dependencies. The triggered error is resolvable by making the method, therefore, the
-            // overall effect 0.
+            // dependencies. The triggered error is resolvable by making the method @Nullable,
+            // therefore,
+            // the overall effect is 0.
```

Comment:
I think GJF messed this comment's formatting for you, please fix :)

Reply:
Thank you for noticing this, fixed it :) 178889458ebbd088bdcc2400eb05e7b58e885b4e